packages/storybook-html/stories/components/header/header.stories.js

```diff
@@ -32,45 +32,39 @@ export const HeaderInterim = {
     const headerElem = canvasElement.querySelector(".uq-header");
     if (headerElem) new header(headerElem);
```

Comment:
```suggestion
    // Early return if uq-header does not exist.
    if (!headerElem) {
      return;
    }
    new header(headerElem);
```

Reply:
The current function will look for the headerElem but when it can't find it continue to the next line and then error as the query selector will be invalid.

So the intent above is to return nothing if the headerElem doesn't exist. 

The issue currently (discussing with @hoffismo ) is that if you include the initialise scripts for accordion / breadcrumb etc ... they would also not run as it has aborted the play script.